packages/react-core/src/components/CalendarMonth/examples/CalendarMonth.md

```diff
@@ -7,6 +7,10 @@ propComponents: ['CalendarMonth', 'CalendarFormat']
 
 ## Examples
 
+When used inline and not contained within a date picker, it's wise to wrap the calendar month component in an HTML 
```

Comment:
Just to link to the date picker in case the calendar month isn't exactly what a user was looking for

```suggestion
When used inline and not contained within a [date picker](/components/date-picker), it's wise to wrap the calendar month component in an HTML 
```